src/components/Channel/Channel.tsx

```diff
@@ -781,54 +785,113 @@ const ChannelInner = <
   const jumpToFirstUnreadMessage = useCallback(
     async (queryMessageLimit = 100) => {
```

Comment:
```suggestion
    async (queryMessageLimit = 100) => {
```
Let's move this default to a constant.

Reply:
Added constant + to other default params in the Channel.tsx file.